### PR TITLE
dev-python/ipython: bump to version 7.13.0

### PR DESCRIPTION
**Pull request by vpayno**
@zmedico @gyakovlev @chutz

### CODE REVIEW

**Comment by vpayno** [comment]
```
--- ipython-7.5.0.ebuild        2020-03-05 11:29:17.068754811 -0800
+++ ipython-7.13.0.ebuild       2020-03-08 13:41:42.840075893 -0700
@@ -3,7 +3,7 @@

 EAPI=7

-PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_COMPAT=( python3_{6,7,8} )
 PYTHON_REQ_USE='readline,sqlite,threads(+)'

 inherit distutils-r1 eutils virtualx
@@ -18,7 +18,7 @@
 IUSE="doc examples matplotlib notebook nbconvert qt5 +smp test"
 RESTRICT="!test? ( test )"

-CDEPEND="
+BDEPEND="
        dev-python/backcall[${PYTHON_USEDEP}]
        dev-python/decorator[${PYTHON_USEDEP}]
        dev-python/jedi[${PYTHON_USEDEP}]
@@ -31,11 +31,10 @@
        matplotlib? ( dev-python/matplotlib[${PYTHON_USEDEP}] )
 "

-RDEPEND="${CDEPEND}
+RDEPEND="${BDEPEND}
        nbconvert? ( dev-python/nbconvert[${PYTHON_USEDEP}] )"

-DEPEND="${CDEPEND}
-       dev-python/setuptools[${PYTHON_USEDEP}]
+DEPEND="${BDEPEND}
        test? (
                >=dev-python/ipykernel-5.1.0[${PYTHON_USEDEP}]
                dev-python/nbformat[${PYTHON_USEDEP}]
@@ -81,7 +80,7 @@
 python_compile_all() {
        if use doc; then
                emake -C docs html_noapi
-               HTML_DOCS=( docs/build/html/. )
+               local -a HTML_DOCS=( docs/build/html/. )
        fi
 }
```

**Comment by vpayno** [comment]
prs created to support this version bump:

https://github.com/gentoo/gentoo/pull/14885 - dev-python/nbformat

https://github.com/gentoo/gentoo/pull/14884 - dev-python/jupyter-client

https://github.com/gentoo/gentoo/pull/14883 - dev-python/noteboot

https://github.com/gentoo/gentoo/pull/14882 - dev-python/ipyparallel

https://github.com/gentoo/gentoo/pull/14880 - dev-python/prompt_toolkit

https://github.com/gentoo/gentoo/pull/14879 - dev-python/traitlets

https://github.com/gentoo/gentoo/pull/14878 - dev-python/nbsphinx

https://github.com/gentoo/gentoo/pull/14877 - dev-python/nbconvert

https://github.com/gentoo/gentoo/pull/14876 - dev-python/pyzmq

https://github.com/gentoo/gentoo/pull/14874 - dev-python/jupyter_core

https://github.com/gentoo/gentoo/pull/14871 - dev-python/ipykernel

File: dev-python/ipython/ipython-7.13.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	dev-python/jedi[${PYTHON_USEDEP}]
+	dev-python/pexpect[${PYTHON_USEDEP}]
+	dev-python/pickleshare[${PYTHON_USEDEP}]
+	>=dev-python/prompt_toolkit-2[${PYTHON_USEDEP}]
+	<dev-python/prompt_toolkit-2.1[${PYTHON_USEDEP}]
+	dev-python/pygments[${PYTHON_USEDEP}]
+	dev-python/traitlets[${PYTHON_USEDEP}]
+	matplotlib? ( dev-python/matplotlib[${PYTHON_USEDEP}] )
+"
+
```
Comment (heroxbd) [inline dev-python/ipython/ipython-7.13.0.ebuild]: I don't think this matches the meaning of `BDEPEND` https://devmanual.gentoo.org/general-concepts/dependencies/index.html

**Comment by vpayno** [comment]
Merging into a single iPython PR.

**Pull request closed without merging**